routing/pathfind.go

```diff
@@ -346,10 +338,13 @@ func newRoute(amtToSend, feeLimit lnwire.MilliSatoshi, sourceVertex Vertex,
 			return nil, newErrf(ErrFeeLimitExceeded, err)
 		}
 
-		// As a sanity check, we ensure that the selected channel has
-		// enough capacity to forward the required amount which
-		// includes the fee dictated at each hop.
-		if nextHop.AmtToForward.ToSatoshis() > nextHop.Channel.Capacity {
+		// As a sanity check, we ensure that the incoming channel has
+		// enough capacity to carry the required amount which
+		// includes the fee dictated at each hop. Make the comparison
+		// in msat to prevent rounding errors.
+		if nextHop.AmtToForward + fee > lnwire.NewMSatFromSatoshis(
+			nextHop.Channel.Capacity) {
```

Comment:
`nextHop.Capacity` should be enough.

Reply:
Same as with edgepolicy, it does not work unless I remove the Channel name.